style.css

```diff
@@ -0,0 +1,800 @@
+/* Global Styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+.la-globe {
+  color: #ec5242;
+}
+
+.logo > a {
+  text-decoration: none;
+  font-size: 24px;
+  color: #272a31;
+  font-weight: 900;
+}
+
+/* Mobile Styles */
+@media screen and (max-width: 768px) {
+  #top-head > ul {
+    display: none;
+  }
+
+  /* About Page Styles Mobile */
+
+  #form {
+    background-color: #272a31;
+    background-image: url('images/pattern.png');
+    background-size: cover;
+    padding-top: 20px;
+    padding-bottom: 20px;
+  }
+
+  main > h1 {
+    color: #ec5242;
+    font-size: 25px;
+  }
+
```

Comment:
- [ ] The mobile designs of your Home and About Pages are not identical to that of the template provided. The hamburger button doesn't show on mobile view and the design in general is different. 👍 It would be nice to style your mobile views properly

### Your Design (Home Page)

![image](https://user-images.githubusercontent.com/77106962/219222525-7ba38f8f-9a99-4061-b9ad-73e295083301.png)

![image](https://user-images.githubusercontent.com/77106962/219222591-4e7f745b-4d22-4b30-9649-7522116a0e59.png)

### Your Design (About Page)

![image](https://user-images.githubusercontent.com/77106962/219222713-0e832151-6f01-498f-a3f3-120caf619bda.png)

![image](https://user-images.githubusercontent.com/77106962/219222767-862456e0-b455-41a5-8e19-ab3663360d60.png)

### Behance Design (Home Page)

![image](https://user-images.githubusercontent.com/77106962/219222879-23bd0b6d-dafe-4cb3-8a90-cbe0d58cbac5.png)

![image](https://user-images.githubusercontent.com/77106962/219222936-10779f1a-5093-4fc1-9d78-0e9faafa2475.png)


### Behance Design (About Page)

![image](https://user-images.githubusercontent.com/77106962/219223072-05202480-c460-44a0-9bc3-0edca4b1d3a7.png)

![image](https://user-images.githubusercontent.com/77106962/219223162-4bb869dc-9c53-4b77-9366-71ccbcc43ca4.png)

![image](https://user-images.githubusercontent.com/77106962/219223201-f79ff767-51dc-4ebf-b13e-cd6e0bc7b07d.png)

#### Hint: Kindly follow the structure for the mobile view on both pages as shown in the template https://www.behance.net/gallery/29845175/CC-Global-Summit-2015